importer/SeBbrSv.py

```diff
@@ -141,9 +139,11 @@ def set_no_of_buildings(self):
         extracted_no = utils.get_number_from_string(
             utils.get_text_inside_brackets(self.funktion))
         if extracted_no is not None:
+            ref = self.wlm_source
             self.add_statement(
                 "has_parts_of_class", "Q41176",
-                {"quantity": {"quantity_value": extracted_no}})
+                {"quantity": {"quantity_value": extracted_no}},
+                refs=[ref])
```

Comment:
Two questions on `add_statement`:
1) Do you ever need more than one reference? (e.g. WikidataStuff does not support it)
2) If you do need multiple are single references more frequent? If so you might consider adding a `listify` call in `add_statements`. I.e. something like:
```
if refs and not isinstance(refs, list):
    refs = [refs, ]
```
or reuse the function for this in WikidataStuff (`helpers.listify()`)